HACKING.rst

```diff
@@ -34,28 +34,40 @@ version by running::
 All that separates now from the fully functioning development environment is a
 few dependencies. We can install them by running the next command::
 
-   (venv) $ pip install \
-              -r sanity.requirements \
-              -r units.requirements \
-              -r integration.requirements
+   (venv) $ make requirements
 
 And this is it. We are all set now. To validate our setup, we can run the
 bundled tests::
 
+   (venv) $ make sanity
+   (venv) $ make units
+   (venv) $ make -j8 integration
+
+We may want to go grab a cup of tea after running that last command, since it
+will take a while (about ten minutes on a semi-decent development machine).
+
+Also, for that last batch of tests, we need to have docker installed,
+configured, and running. And that is it. If nothing broke, we are ready to
+start developing.
+
+
+Section with content that will be reworked into a development story
+-------------------------------------------------------------------
+
+TODO: Describe the process of adding a new module.
+
    (venv) $ ansible-test sanity --python 3.7
```

Comment:
With the latest changes, this can be reduced to `$ make sanity`. 

EDIT: From the name of the section, I see that this content is subject to further polishing in the near future, but I think it makes sense to modify this here and now, since we now have more powerful `Makefile`s.